tests/test_vtk.py

```diff
@@ -45,6 +59,97 @@ def test_pluto_units(pluto_vtk_file):
         assert_allclose_units(getattr(ds, f"{u}_unit"), expected)
 
 
+def test_pluto_complete_units_override(pluto_vtk_file):
+    su = SAMPLE_UNITS
+    unit_combs = distinct_combinations(su, 3)
+    for comb in unit_combs:
+        uo = {}
+        for unit in comb:
+            uo[unit] = su[unit]
+        if set(uo) in PlutoVtkDataset.invalid_unit_combinations:
+            continue
+        ds = yt_idefix.load(
+            pluto_vtk_file["path"],
+            geometry=pluto_vtk_file["geometry"],
+            units_override=uo,
+        )
+        assert_allclose_units(ds.length_unit, ds.quan(*su["length_unit"]))
+        assert_allclose_units(ds.mass_unit, ds.quan(*su["mass_unit"]))
+        assert_allclose_units(ds.time_unit, ds.quan(*su["time_unit"]))
+        assert_allclose_units(ds.density_unit, ds.quan(*su["density_unit"]))
+        assert_allclose_units(ds.velocity_unit, ds.quan(*su["velocity_unit"]))
+        assert_allclose_units(ds.magnetic_unit, ds.quan(*su["magnetic_unit"]))
+
+
+def test_pluto_one_unit_override(pluto_vtk_file):
+    # Pluto's length_unit and density_unit will be combined with
+    uo = {"time_unit": (2.0, "yr")}
+    ds = yt_idefix.load(
+        pluto_vtk_file["path"], geometry=pluto_vtk_file["geometry"], units_override=uo
+    )
+    expect_velocity = ds.length_unit / ds.quan(*uo["time_unit"])
```

Comment:
```
expect_velocity = plut_vtk_file["units"]["length"] / ds.quan(*uo["time_unit"])
```
Maybe this is better. My intention is to check if right Pluto's unit was preserved and participated in unit derivation. The value of `expect_velocity` is different for each dataset because they have different defined units. It also depends on the unit in `uo` because if we give `length_unit` in `uo`, `density_unit` and `velocity_unit` will be preserved instead of `density_unit` and `length_unit` here. So I don't think it's a good idea to hardcode the expected value in `meta.yaml`.

Reply:
Ok that makes sense